jazz_delete-serverless-service/config/validTest-config.json

```diff
@@ -1,5 +1,14 @@
 {
```

Comment:
This will lead to large number of config files sitting along with stg/prod/dev version of config files. Can we not create/override configurations specific to tests within the test?

Reply:
Keeping this list to dev/stg/prod/test-config.json is easier to manage/understand than expanding this to dev/stg/prod/test/valid-test/random-blah-test/someother-test-config.json.

Reply:
The usual model is that we have "test-config" which is used as base config and as a `setup` for your test. In this case looks like we went away from that pattern. We will add a story to keep it consistent with the rest of the code base.